monkey/monkey_island/cc/services/attack/technique_reports/T1065.py

```diff
@@ -14,6 +14,9 @@ class T1065(AttackTechnique):
 
     @staticmethod
     def get_report_data():
-        port = ConfigService.get_config_value(CURRENT_SERVER_PATH).split(":")[1]
+        monkey = Monkey.objects.first()
+        tunnel = monkey.get_tunnel_info()["tunnel"]
+        port = tunnel.split(":")[1] if tunnel is not None else ISLAND_PORT
```

Comment:
This will raise `IndexError` if the tunnel is malformed and does not contain a ":".

Reply:
It still raises an `IndexError` if the tunnel address is malformed. If we're putting a domain model in place, I suppose this will resolve itself.